opensearch/src/test/java/org/opensearch/sql/opensearch/executor/OpenSearchExecutionEngineTest.java

```diff
@@ -61,6 +61,10 @@ class OpenSearchExecutionEngineTest {
 
   @Mock private Split split;
 
+  @Mock private static OpenSearchIndexScan fakeIndexScan;
+
+  @Mock private static ProjectOperator fakeProjectOperator;
```

Comment:
Why static?

Reply:
Fixed in b3e5daed13712d5e2475d036d6c557eaa242d816